.github/workflows/test.yml

```diff
@@ -0,0 +1,90 @@
+name: Timber starter theme tests
+
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
```suggestion
  push:
    branches:
      - '1.x'
      - '2.x'
```

It will prevent double runs.

Reply:
Alright, let’s try it out! Thanks for looking into the details 😎.

Reply:
22 👉🏻 11